init-tools.sh

```diff
@@ -46,18 +46,18 @@ if [ ! -e $__DOTNET_PATH ]; then
         case $OSName in
             Darwin)
                 OS=OSX
-                __DOTNET_PKG=dotnet-dev-osx-x64
+                __DOTNET_PKG=dotnet-sdk-${__DOTNET_TOOLS_VERSION}-osx-x64
                 ulimit -n 2048
                 ;;
 
             Linux)
-                __DOTNET_PKG=dotnet-dev-linux-x64
+                __DOTNET_PKG=dotnet-sdk-${__DOTNET_TOOLS_VERSION}-linux-x64
                 OS=Linux
 
                 if [ -e /etc/redhat-release ]; then
                     redhatRelease=$(</etc/redhat-release)
                     if [[ $redhatRelease == "CentOS release 6."* || $redhatRelease == "Red Hat Enterprise Linux Server release 6."* ]]; then
-                        __DOTNET_PKG=dotnet-dev-rhel.6-x64
+                        __DOTNET_PKG=dotnet-sdk-${__DOTNET_TOOLS_VERSION}-rhel.6-x64
```

Comment:
@janvorli - do we have a `2.0.0` rhel.6 SDK?  I assume 'no'.

Reply:
We do. I needed to create that as a new bootstrap cli when corefx was migrated to 2.0.0

Reply:
Awesome!  Then this works.